### ISSUE TITLE
"Too much recursion"

### ISSUE THREAD

**Issue by alexolog** [Tampermonkey/tampermonkey#1466]
## Expected Behavior
TM beta should behave normally, like the stable version (6154)

## Actual Behavior
After latest beta update on FF, noticing a long delay in loading sites, some scripts no longer work, messages in the log:
```
Unchecked lastError value: Error: too much recursion common.js:4
    sendMessage moz-extension://eb61ac3e-ed19-4b96-93b1-d906591e63a5/rea/common.js:4
```

## Specifications

- FF 99.0.b8
- TM v4.16.6158
- OS windows 10

**Comment by derjanb** [comment]
Hmm, I can not reproduce this. 
Can you please add mroe information? What script and page? Can you please check with a new temporary profile? Does [Tampermonkey BETA](https://www.tampermonkey.net/?browser=firefox) show the same issue?

**Comment by alexolog** [comment]
This only happens with the beta

**Comment by derjanb** [comment]
Duplicate of #1473

**Comment by JamYiz** [comment]
> Duplicate of #1473
I also have this problem in Firefox, please fix it in time

Uncaught (in promise) InternalError: too much recursion

**Comment by derjanb** [comment]
Can you please check whether the issue is fixed at 4.17.6161 ([xpi](https://firefox.tampermonkey.net/firefox-current-beta.xpi))?

Please install the [BETA version](https://www.tampermonkey.net/index.php?browser=firefox) or check for BETA version updates at `about:addons` 

To test the fix please export your settings and scripts as zip or (JSON) file at the "Utilities" tab and import it back at the fixed BETA version.
